utility/environment_check.py

```diff
@@ -23,12 +24,32 @@
 SECRET = ocp.OCP(kind=constants.SECRET)
 NS = ocp.OCP(kind=constants.NAMESPACE)
 
+KINDS = [POD, SC, CEPHFILESYSTEM, CEPHBLOCKPOOL, PV, PVC, SECRET, NS]
+ENV_STATUS_PRE = {
+    'pod': None,
+    'sc': None,
+    'cephfs': None,
+    'cephbp': None,
+    'pv': None,
+    'pvc': None,
+    'secret': None,
+    'namespace': None,
+}
+ENV_STATUS_POST = {
```

Comment:
I see that this is just duplication of the code above. What about to have it defined as ENV_STATUS_DICT and then
```python
ENV_STATUS_DICT = {
    'pod': None,
    'sc': None,
    'cephfs': None,
    'cephbp': None,
    'pv': None,
    'pvc': None,
    'secret': None,
    'namespace': None,
}


ENV_STATUS_PRE = ENV_STATUS_DICT.copy()
ENV_STATUS_POST = ENV_STATUS_DICT()
```

If you don't have nested mutable structure yet as you referring all to None here I think you can just use copy here.
Was thinking about problems with mutable dict here but you are all the time gather data to those nested structures, you basically overwrite it all the time so shouldn't be an issue here.

Reply:
changed to:

```python
ENV_STATUS_DICT = {
    'pod': None,
    'sc': None,
    'cephfs': None,
    'cephbp': None,
    'pv': None,
    'pvc': None,
    'secret': None,
    'namespace': None,
}
ENV_STATUS_PRE = copy.deepcopy(ENV_STATUS_DICT)
ENV_STATUS_POST = copy.deepcopy(ENV_STATUS_DICT)
```